client/app/styling/app.css

```diff
@@ -0,0 +1,136 @@
+.main {
```

Comment:
Having all the css in one file takes away some of the use cases of CSS modules I think. You have to be wordier with the class names. I guess that's not such a big deal though. I'd be curious actually if there's no difference between this and separate files if the components themselves are in separate files (like they should be, and how you have them). Do you know how that works? I wonder where exactly the modularity is implemented. Just thinking out loud haha!